.github/workflows/release.yml

```diff
@@ -52,3 +55,16 @@ jobs:
           refreshToken: ${{ secrets.OAUTH_REFRESH_TOKEN }}
           force: true # We have checked the `dart pub publish --dry-run` `in build.yaml`, it's ok to force publish here.
           skipTests: true
+      - name: Build DartDoc 📖
+        run: |
+          # Generate DartDoc
+          dart doc
+
+          # Create a zip file of the DartDoc output
+          zip -r flutter-agora-docs.zip doc
+      - name: Upload DartDoc Archive to GitHub release ⬆️
+        uses: svenstaro/upload-release-action@2.6.0
+        with:
+          file: flutter-agora-docs.zip
```

Comment:
I suggest renaming it to `agora_rtc_engine_docs` to better match the package name.